src/components/Organisms/Header/Nav/Nav.js

```diff
@@ -57,14 +53,10 @@ const MainNav = ({ navItems }) => {
   };
 
   useEffect(() => {
-    const isTouchDevice = (typeof window !== 'undefined' && 'ontouchstart' in window)
-      || window.navigator.msMaxTouchPoints > 0;
-
     const width = typeof window !== 'undefined' ? window.innerWidth : null;
```

Comment:
Not anything you've added but could remove this check while we're here as `useEffect` only ever runs in the browser